acrg/paris_formatting/make_paris_outputs.py

```diff
@@ -115,24 +115,33 @@ def make_country_output(
         "NW_EU_CONTINENT": "BEL-DEU-FRA-LUX-NLD",
     }
 
-    region_traces = []
-    for region, countries_str in regions_dict.items():
-        region_countries = countries_str.split("-")
-        region_ds = (
-            country_traces_merged.sel(country=region_countries).sum("country").expand_dims({"country": [region]})
+    try:
+        region_traces = []
+        for region, countries_str in regions_dict.items():
+            region_countries = countries_str.split("-")
+            region_ds = (
+                country_traces_merged.sel(country=region_countries).sum("country").expand_dims({"country": [region]})
+            )
+            region_traces.append(region_ds)
+
+        region_traces_merged = xr.concat(region_traces, dim="country")
+
+        # combine country and region traces
+        all_traces_merged = xr.merge([country_traces_merged, region_traces_merged])
+
+        country_output = xr.merge(
+            calculate_stats(
+                all_traces_merged, "country", chunk_dim="country", chunk_size=1, report_mode=report_mode
+            )
         )
-        region_traces.append(region_ds)
+    except Exception as e:
```

Comment:
I think if a country is missing, the error it should cause is `KeyError`. It's good to try to catch something specific, since you don't want to ignore other errors necessarily. (In this case, it probably doesn't matter too much.) Anyway, a `KeyError` would mean that one of the countries isn't present in the `country` coordinate.